CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ and this project adheres to Rust's notion of
   - `FieldExt::from_u64` (use `From<u64> for ff::PrimeField` instead).
   - `FieldExt::{from_bytes, read, to_bytes, write}`
     (use `ff::PrimeField::{from_repr, to_repr}` instead).
+  - `FieldExt::rand` (use `ff::Field::random` instead).
 
```

Comment:
Also document the changes to `std` gating.

Reply:
We've not released a crate version with the `std` feature flag, so to crate users they'll see the addition of no-std support. Agreed on documenting the new flags, but I'll do that in a separate PR once everything is stable.